assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml

```diff
@@ -669,9 +669,9 @@ jobs:
       model_selector_output: '${{parent.jobs.question_answering_model_import.outputs.output_dir}}'
     outputs:
       pytorch_model_folder: '${{parent.outputs.pytorch_model_folder}}'
-  ft_nlp_model_converter:
+  question_answering_model_converter:
     type: command
-    component: azureml:ft_nlp_model_converter:0.0.53
+    component: azureml:question_answering_model_converter:0.0.54
```

Comment:
you need to update the model path at evaluation component

Reply:
Fixed, Thanks SitaRam